app/build.gradle

```diff
@@ -4,21 +4,6 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 apply plugin: 'kotlin-kapt'
 
-/*
- * Enable the application to consume the Video Android SDK directly from source
- * instead of from Bintray.
- *
- * The property is not defined by default when consuming the
- * application as a stand-alone project, but is defined by default in the Video Android SDK project.
- * This property enables the Video Android SDK project to build the application as a git subtree
- * with the contents of the source itself as a dependency. This coupling is intentional as it allows
- * SDK developers to iterate on SDK features within the application that is maintained in a separate
- * project.
- */
-def videoAndroidSdkDepProp = "videoAppSdkDependency"
```

Comment:
Please see the app subtree in the SDK for how to import the SDK so that.

https://github.com/twilio/twilio-video-android/blob/master/videoapp/app/build.gradle

Reply:
@aaalaniz Yeah I've noticed that this file is out of sync. I'll look at the subtree `videoapp/build.gradle` file and try to make the app repo's version match it.